### PR TITLE
Create static version of new standard footer

### PR DESCRIPTION
**Pull request by jwade1327**

### CODE REVIEW

**Comment by jwade1327** [comment]
Screen Shot 2021-08-11 at 11 25 43 AM

**Comment by jwade1327** [comment]
@brandonbk screenshot added

File: packages/common/components/blocks/new-standard-footer.marko [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      </tr>
+      <tr>
+        <td align="center" valign="top"  class="align">
+          <table width="242" align="center"  class="align" border="0" cellspacing="0" cellpadding="0">
+            <tr>
+              <td align="center" valign="middle"><img src="https://wfh0621.upupload.com/71644/rx1f7he0s271644/images/fb.png" alt="fb" width="35" height="34"></td>
+              <td align="center" valign="middle"><img src="https://wfh0621.upupload.com/71644/rx1f7he0s271644/images/tw.png" alt="tw" width="34" height="28"></td>
+              <td align="center" valign="middle"><img src="https://wfh0621.upupload.com/71644/rx1f7he0s271644/images/insta.png" alt="insta" width="35" height="34"></td>
+              <td align="center" valign="middle"><img src="https://wfh0621.upupload.com/71644/rx1f7he0s271644/images/you.png" alt="you" width="35" height="24"></td>
+              <td align="center" valign="middle"><img src="https://wfh0621.upupload.com/71644/rx1f7he0s271644/images/link.png" alt="link" width="35" height="34"></td>
```
Comment (solocommand) [inline packages/common/components/blocks/new-standard-footer.marko:15]: These files should be moved to S3

**Review by solocommand — CHANGES_REQUESTED** [review]

**Comment by jwade1327** [comment]
These images are now added to S3 @solocommand

**Comment by solocommand** [comment]
@jwade1327 looks good except the hostname! This should pull the image domain from the site (I'm sure we've done this on other static header images/etc elsewhere), resulting in `https://img.forconstructionpros.com/files/...` on FCP instead of `https://s3.amazonaws.com/media.base.parameter1.com/files/...`.

If you can't find an example let me know and I can hunt one down

**Comment by jwade1327** [comment]
@solocommand ok now I finally got it!

**Comment by brandonbk** [comment]
@jwade1327 - this is fine for this first pass, since its intended to be static.  I was going to ask you to make the image host (img.sdcexec.com) dynamic or config-driven, so it wasn't forced to sdc's value, but that can come in a later pass when all of the social icons become config-driven.

**Pull request merged**